### ISSUE TITLE
testflow package md5sum failed.

### ISSUE THREAD

**Issue by Valerio-Perez** [hilanderas/routing#2]
# Issue description
testflow package md5sum failed.

### Test environment
* Package and version : routing-testflow-0.0.11.zip

### How to reproduce
```bash
wget https://github.com/hilanderas/routing/releases/download/0.0.11/routing-testflow-0.0.11.zip
unzip routing-testflow-0.0.11.zip
cd routing-testflow
md5sum -c routing-testflow-0.0.11.md5
```

### Actual results
```
./.env: FAILED
./Makefile: OK
./state.mk: OK
./.routing.env: OK
./basic.mk: OK
./function.mk: OK
./routing.mk: OK
md5sum: WARNING: 1 computed checksum did NOT match
```

### Expected results
I expect that all files is ok when checking md5

**Comment by Valerio-Perez** [comment]
Find this issue location:
* worry order of target `build-testflow` in Makefile(specify with `**`):
```makefile
...
build-testflow:
	cp -r testflow/script $(TESTFLOW)
	**cd $(TESTFLOW)/; find . -type f -exec md5sum {} \; > $(CURDIR)/$(TESTFLOW)-$(version).md5; cd -**
	mv $(TESTFLOW)-$(version).md5 $(TESTFLOW)
	cd $(TESTFLOW) && make -f basic.mk set_mod TESTMODE=prod
	**sed -i '/PROJ_VERSION/c\export PROJ_VERSION=${version}' $(TESTFLOW)/.env **
	zip -r $(TESTFLOW)-$(version).zip $(TESTFLOW)
	rm -rf $(TESTFLOW)
...
```

**Comment by Valerio-Perez** [comment]
This has been resolved in [0.0.12](https://github.com/hilanderas/routing/releases/tag/0.0.12)
